functions/Invoke-DbaQuery.ps1

```diff
@@ -118,7 +121,8 @@ function Invoke-DbaQuery {
         [switch]$MessagesToOutput,
         [parameter(ValueFromPipeline)]
         [Microsoft.SqlServer.Management.Smo.Database[]]$InputObject,
-        [switch]$EnableException
+        [switch]$EnableException,
+        [switch]$ReadOnly
```

Comment:
`EnableException` should always be the last parameter, just for tab purposes.

Reply:
done, I'll be pushing the change with the few more updates shortly